src/main/webapp/app/course/shared/course-sidebar-toggle-button/course-sidebar-toggle-button.component.html

```
@@ -0,0 +1,16 @@
+<button
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)